components/footers/PageFooter.vue

```diff
@@ -7,11 +7,11 @@
             Qiskit Elements
           </h2>
           <ul>
-            <li><a class="footer-column__link" href="https://qiskit.org/terra">Terra</a></li>
-            <li><a class="footer-column__link" href="https://qiskit.org/aer">Aer</a></li>
-            <li><a class="footer-column__link" href="https://qiskit.org/aqua">Aqua</a></li>
-            <li><a class="footer-column__link" href="https://qiskit.org/ignis">Ignis</a></li>
-            <li><a class="footer-column__link" href="https://qiskit.org/ibmqaccount">IBM Q Account</a></li>
+            <li v-for="element in elements" :key="element.label" >
```

Comment:
According to [Vue documentation](https://vuejs.org/v2/api/#key): "Children of the same common parent must have unique keys". Although likely, we cannot ensure the `element.label` will be different. Since the elements are sorted, we can use the position of the element in the array. This is done by iterating the array in a slightly different way:

```suggestion
            <li v-for="(element, index) in elements" :key="index" >
```

Reply:
According also to [Vue documentation](https://vuejs.org/v2/api/#key): "The key special attribute is primarily used as a hint for Vue’s virtual DOM algorithm to identify VNodes when diffing the new list of nodes against the old list." These list aren't mutable and the elements of the list have unique labels, so I think that it fit with the vue documentation. I can change it if you prefere to have `index`, but we have to keep in mind that in the case that the list were mutable, using the `index` as `key` is a [bad solution](https://stackoverflow.com/questions/44531510/why-not-always-use-the-index-as-the-key-in-a-vue-js-for-loop)

Reply:
I'm fine with your defensive approach. It is unlikely and probably an error if two labels are the same.